news/2022-09-01-how-zetera-became-a-medal-hunter.md

```diff
@@ -0,0 +1,79 @@
+
+The limits of the medal system needed to be examined more closely, albeit with educated guesswork and deduction. Surely it can't check when you lost your full combo or when an SS was lost, so the only things left would be the miss count, max combo, counts of 300s, 100s, 50s and misses, playing in different modes and a couple of other things, such as at what time was the score submitted. At the time, the flair text read as "First Generation in more than one ways", which lead Zetera to think that it might be related to Pokémon or perhaps the first generation of influential mappers, but in the end, found himself stumped.
+
+Out of ideas, he reached out to [Ephemeral](https://osu.ppy.sh/users/102335), osu!'s "community manager", and instead of receiving a hint, Ephemeral would go on to create the [following thread](https://osu.ppy.sh/community/forums/topics/662279?n=1) on the osu! forum:
+
+> It's been roughly a week since the Halloween medal release, and just about every medal outside of one has been conclusively claimed and figured out.
+>
+> Originally, I thought that the Firmament Moves would be the baffler for most people. Apparently not!
+>
+> So, let's have a little fun with this, and up the ante a little bit.
+
+And "up the ante a little bit" he did. A bounty was set, stating that the first person who unlocks the medal and messages Ephemeral with the correct set of criteria required to unlock it would receive a "Medal Hunter" user title, which would the first of its kind, and to this day, remains as such.
+
+The post also shed some light on how the new medal system works and what the loose criteria would be for not just The Girl in the Forest, but all medals included in the Halloween update and going forward. Following this, Zetera descended into a maddening 5-hour search of trial and error. Maybe the max combo or accuracy needed to match numbers relevant to the first generation of Pokémon games? Maybe the accuracy needed to match the release date of the first game? No, the HP drain was too high to achieve a ~19% accuracy run. "How many Pokémon were in the first game?" he thought, maybe there needs to be a certain amount of 300s, 100s and 50s...
+
+After a gruelling slog of trial and error, he finally found himself staring at the fruits of his labor, and after sending a forum PM to Ephemeral with the correct answer, he was officially dubbed as osu!'s first Medal Hunter. He was really happy about receiving the medal, but he felt like there were other people in the community that deserved this medal more than him, and had to say this during our chat:
+
+> User titles and I are a complicated topic. It's not really part of this interview's topic, but I did a lot of different things for the community in the hopes of leaving a mark somewhere(...), because if there was something I wanted more than anything else, it was the Elite Mapper title.
```

Comment:
```suggestion
> User titles and I are a complicated topic. It's not really part of this interview's topic, but I did a lot of different things for the community in the hopes of leaving a mark somewhere [...] because if there was something I wanted more than anything else, it was the Elite Mapper title.
```